troubleshooting/Shopify/inventory.md

```diff
@@ -0,0 +1,39 @@
+#  Inventory Auditing
+
+**Scenario:** Inventory Not Synced from HotWax Commerce to Shopify
+
+Sometimes retailers may encounter inventory disparities between HotWax Commerce and Shopify Inventory. This discrepancy could result in inventory appearing out of stock in Shopify despite being available in HotWax Commerce.
+
+## Steps to Troubleshoot
+
+### Verify in HotWax Commerce
+
+1. **Check Job Status:**
+
+   - Navigate to the `HotWax Commerce Job Manager App`.
+   - Locate the job named `Upload recent inventory change`. This job is responsible for updating all inventory changes in Shopify that have occurred in HotWax Commerce since the last time the job ran.
```

Comment:
As both of these jobs are responsible for inventory changes, we should write in that way and ask user to verify on basis of its use case.